client/dist/styles/product_overview.css

```diff
@@ -275,4 +275,16 @@
   border-radius: 50%;
   border: solid .02em;
   background-color: white;
+}
+
+.please_select_size {
```

Comment:
if you have two different classes doing the same thing you can do this .please_select_size, .sale_price {color: red} to save a line

Reply:
you are right! that will be my tech debt, I'm sure I have lots of refactoring to do, thanks1